examples/maytrig/ex_maytrig.cpp

```diff
@@ -0,0 +1,80 @@
+#include "daisysp.h"
+#include "daisy_seed.h"
+
+// Shortening long macro for sample rate
+#ifndef sample_rate
+
+#endif
+
+// Interleaved audio definitions
+#define LEFT (i)
+#define RIGHT (i+1)
+
+using namespace daisysp;
+using namespace daisy;
+
+static DaisySeed seed;
+static AdEnv env;
+static Oscillator osc, tick;
+static Maytrig maytrig;
+
+static void AudioCallback(float *in, float *out, size_t size)
+{
+	float osc_out, env_out;
+    for (size_t i = 0; i < size; i += 2)
+    {
+        // When the metro ticks, trigger the envelope to start.
+        if (tick.Process() >= .9999)
```

Comment:
I'd replace your Oscillator `tick` with `Metro` object as it is more suited to what you're trying to do here.